libiberty/strerror.c

```diff
@@ -460,12 +460,20 @@ static int num_error_names = 0;
 static int sys_nerr;
 static const char **sys_errlist;
 
+#else
+#ifdef DARWIN
```

Comment:
```suggestion
#elif defined(DARWIN)
```
Would remove the need for an extra trailing `#endif`